docs/setup/index.md

```diff
@@ -27,7 +27,7 @@ Refer to the following table to select the appropriate installation method for y
 | :simple-linux: Ubuntu 20 and 22 (LTS)             | [:white_check_mark:][conda_page] | [:white_check_mark:][pip_page] | [:white_check_mark:][notebooks_page] | [:white_check_mark:][nocode]          | [:white_check_mark:][kni] |
 | :simple-linux: Debian 10                          | [:white_check_mark:][conda_page] | [:white_check_mark:][pip_page] | [:white_check_mark:][notebooks_page] | [:white_check_mark:][nocode]          | [:white_check_mark:][kni] |
 | :simple-linux: Debian 11 and 12                   | [:white_check_mark:][conda_page] |                                | [:white_check_mark:][notebooks_page] |                                       | [:white_check_mark:][kni] |
-| :simple-linux: Rocky Linux 9                      | [:white_check_mark:][conda_page] | [:white_check_mark:][pip_page] | [:white_check_mark:][notebooks_page] |                                       | [:white_check_mark:][kni] |
+| :simple-linux: Rocky Linux 8 and 9                      | [:white_check_mark:][conda_page] | [:white_check_mark:][pip_page] | [:white_check_mark:][notebooks_page] |                                       | [:white_check_mark:][kni] |
```

Comment:
I would put an asterisk to Rocky 8 stating that a supported Python version (>= 3.8) needs to be installed first (indeed, vanilla Rocky 8 has Python 3.6).

Reply:
I think this part is already loaded with many details for specific systems. I would avoid adding a new asterisk especially since it targets an "advanced" installation option.

Reply:
OK. However, for Rocky 8 specifically, I would just re-remember the range of supported Pythons and state that the user installing on Rocky 8 should make sure one of the supported Pythons is installed.